.github/workflows/buf-lint.yaml

```
@@ -0,0 +1,20 @@
+name: Buf Lint
+
+on: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)